girder/cli.py

```diff
@@ -0,0 +1,15 @@
+from click_plugins import with_plugins
+from pkg_resources import iter_entry_points
+import click
+
+
+@with_plugins(iter_entry_points('girder.cli_plugins'))
+@click.group(help='Girder: data management platform for the web.',
+             context_settings=dict(help_option_names=['-h', '--help']))
```

Comment:
Could you add a comment, just explaining that the point of this is to add an `-h` option, which is not there by default? It's not obvious from the source.

Reply:
`help_option_names=['-h', '--help'])` as an input to a CLI command is as obvious as it gets (without adding a comment).